packages/runtime/test-runtime-utils/package.json

```diff
@@ -64,17 +64,16 @@
     "@fluidframework/routerlicious-driver": "^0.53.0",
     "@fluidframework/runtime-definitions": "^0.53.0",
     "@fluidframework/runtime-utils": "^0.53.0",
+    "@fluidframework/server-services-client": "^0.1034.0-0",
```

Comment:
fyi the change to remove prereleases after release is in-flight right now

Reply:
I'll do a merge with remote main once it is finished and prior to merging this in

Reply:
i'm not sure this is a coupling we want. i'm also not sure that package is iso-morphic

Reply:
@anthony-murphy where should the code for token generation live then? It is something that might be used both on the client side (i.e. InsecureTokenProvider) or on a node.js service instance (i.e. an Azure Function to securely provide tokens for FRS). I saw a note above the `generateToken` function in this package that said this should be the library that should be used in all other places in the FF repo for generating JWT tokens

Reply:
is there value in sharing this code? or should we just copy it here. that may be cheaper that adding another dependency between server and client

Reply:
on a brief inspection i see there are already 3 copies of this function, including in local driver, probably to avoid just such a dependency.

the function is pretty trivial. so i don't see a huge problem duplicating it again, or maybe there is a single client side, and single server-side place it could live.

Reply:
ideally i love it if there were no server dependencies at all in client, and all interactions were mediated through our common definition packages

Reply:
There is also benefit to duplicating code like this, as if the server changes format, we don't automatically pick it up, and our tests break. which matches the real world where server and client version are decoupled.

Reply:
@anthony-murphy updated with a matching copy on the client-side package azure-service-utils

Reply:
My issue with duplication is that the _reason_ for duplication is often lost to the sands of time, and future people will puzzle over the minor differences between the copies and wonder, "Do these differences _matter_?" So -- can we document this decision somehow in a comment around the duplicate code? The goal is only to help give future people context.

Reply:
Sure, but just to add some context, we're going to have one copy in the client code and one copy in the service code. The other copies of the function that exist in the service packages will need to be converged to use the one service copy

Reply:
Ahhh, I think I misunderstood. Thanks for clarifying! One per client/server makes sense.